ci-operator/config/openshift-priv/installer/openshift-priv-installer-main.yaml

```diff
@@ -675,6 +675,15 @@ tests:
     cluster_profile: equinix-edge-enablement
     workflow: baremetalds-two-node-arbiter
   timeout: 5h0m0s
+- always_run: false
```

Comment:
Do you have a reason to add the jobs to `openshift-priv/installer`? I'm not sure exactly what that is, but I think it's used by QE. If you have a reason to add to priv, no problem for me, but if you just want presubmits on PRs for devs, `openshift/installer` should be sufficient.

Reply:
That's a good point, I'll remove them for now because I'm not sure they are needed yet.